assemblyline/odm/models/ontology/results/malware_config.py

```diff
@@ -9,6 +9,11 @@
               "reverse_proxy", "rootkit", "scanner", "scareware", "spammer", "trojan", "virus", "wiper",
               "webshell", "worm"]
 CONNECTION_USAGE = ["c2", "upload", "download", "propagate", "tunnel", "other", "ransom", "decoy"]
+DNS_RECORD_TYPES = ["A", "AAAA", "AFSDB", "APL", "CAA", "CDNSKEY", "CDS", "CERT", "CNAME", "CSYNC", "DHCID", "DLV",
```

Comment:
duplicate of https://github.com/CybercentreCanada/assemblyline-base/blob/9ed6eb059026ba781f9244ffc390ed38591ab28b/assemblyline/odm/models/ontology/results/network.py#L24